.github/workflows/server.yaml

```diff
@@ -0,0 +1,42 @@
+name: Server
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    name: Server build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.13.1' # The Go version to download (if necessary) and use.
+      - name: Get current Go version
+        run: go version
+      - name: Get Go dependencies
+        working-directory: ./server/
+        run: go mod download
+      - name: Build app
+        working-directory: ./server/
+        run: go build -o ptrun-server ./cmd/ptrun-server/main.go
+  docker:
```

Comment:
Can we make this run only on `develop` & `master` branches as a manual action so we don't push images to ghcr every time someone makes a PR and/or change there.